doc/topics/windows/windows-package-manager.rst

```diff
@@ -916,21 +899,16 @@ files for other types of .exe based installers.
 
 Parameters
 ==========
+This section describes the parameters placed under the ``version``in the package definition file.
+Example can be found on the `Salt winrepo repository <https://github.com/saltstack/salt-winrepo-ng>`_
```

Comment:
**Nitpick:** The ``version`` needs a space after it. For the sentence right after it, change it to "An example can be found..."

Reply:
Updated